extension/platforms/chrome/manifests/manifest.base.json

```diff
@@ -32,7 +32,8 @@
     "storage",
     "tabs",
     "scripting",
-    "contextMenus"
+    "contextMenus",
+    "downloads"
```

Comment:
Looks legit but have we checked we really need this new permission? 
Asking cause validation will take longer on the store (which is okay) and I'm not sure how will auto-update work if we ask for a new permission (maybe it's smooth but I'm unsure). 
🙏🏻